e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -9,6 +9,7 @@ Commands:
   code-pushup autorun       Shortcut for running collect followed by upload
   code-pushup collect       Run Plugins and collect results
   code-pushup upload        Upload report results to the portal
+  code-pushup history       Create history of commits
```

Comment:
```suggestion
  code-pushup history       Collect reports for commit history
```

Reply:
The suggestion is marked as resolved, but the description wasn't changed. Could you update it please? "Create history of commits" sounds a like a description of `git log`, it's not clear that `history`'s running `code-pushup` on those commits. Obviously, it needs to be changed in the source code (`packages/cli/src/lib/history/history-command.ts`) as well as in this snapshot.

![image](https://github.com/code-pushup/cli/assets/34691111/5b28fb08-2e4e-49ae-a6a3-b4bc320b89a5)

Reply:
Thanks! Got confused when merging main and lost this comment. I unfortunately went with the wrong choice then. 

Updated it in the test as well as source code. 
https://github.com/code-pushup/cli/pull/273/commits/ea9083a5a78d4ebcffe1e50c078a9a4f6bbf8b64